app/assets/stylesheets/snowcrash/overrides/_bootstrap_and_override.scss

```diff
@@ -79,3 +79,13 @@ select {
     @include button($orange);
   }
 }
+
+.table thead:first-child tr:first-child th {
```

Comment:
@MattBudz does this css belong to https://github.com/dradis/dradis-ce/blob/master/app/assets/stylesheets/shared/items_table.scss ?